app/Models/User.php

```diff
@@ -46,12 +55,13 @@ class User extends Authenticatable implements Camelizable
      * @var array
      */
     protected $hidden = [
-        'password', 'remember_token',
+        'password',
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "password" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)